cmd/storage/cli.go

```diff
@@ -19,17 +19,17 @@ func treeAction(c *cli.Context) error {
 	}
 	owner, name, rev, path := args[0], args[1], args[2], args[3]
 
-	s, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
+	stor, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
 	if err != nil {
 		return err
 	}
 
-	r, err := s.GetRepository(context.Background(), owner, name)
+	repo, err := stor.GetRepository(context.Background(), owner, name)
```

Comment:
I don't really see a reason to rename this.
```suggestion
	r, err := stor.GetRepository(context.Background(), owner, name)
```

Note: I would prefer spelling it out as `repository` over `repo`. :relaxed:

Reply:
`repository` overlaps with `pkg/repository` hence why I opted for `repo` instead.